ionoscloud/resource_server.go

```diff
@@ -182,7 +182,7 @@ func resourceServer() *schema.Resource {
 			},
 			"volume": {
 				Type:     schema.TypeList,
-				Required: true,
+				Optional: true,
```

Comment:
@cristiGuranIonos shouldn't we add a `ForceNew: true` here?

Reply:
it has inner fields that require forcenew. we don't want to do it for all fields of the volume.